lib/globalize.rb

```diff
@@ -51,6 +51,15 @@ def fallbacks(for_locale = self.locale)
     def default_fallbacks(for_locale = self.locale)
       i18n_fallbacks? ? I18n.fallbacks[for_locale] : [for_locale.to_sym]
     end
+    
+    def fallbacks_for_empty_translations=(value = false)
+      return if !!value != value # Only allow booleans
```

Comment:
I personally find `[true, false].include?  value` more readable and you can also get rid of the comment explaining what it does